.github/workflows/ci.yml

```diff
@@ -36,14 +36,17 @@ jobs:
           - ubuntu-latest
           - macOS-latest
           - windows-latest
-        arch:
-          - x64
+        exclude:
+          - github-runner: macos-latest # Apple Silicon
+            version: '1.3'
+        include:
+          - github-runner: macos-13 # Intel
```

Comment:
Ugh, I didn't realise this was wrong: it's not that CI didn't fail, it didn't run at all.